foxbroadcasting/react-select
# [DPP-1231] Polish Unique Select styles

## What did you implement:

Implementing issue: [DPP-1231](https://fng-jira.fox.com/browse/DPP-1231)

## How did you implement it:
Update the styles for a better user experience

## Files to review:
Please review the following files and ignore the rest because they are auto generated by `nps`
1. `less/control.less`
1. `less/select.less`
1. `src/Async.js`
1. `src/Select.js`
1. `src/utils/defaultArrowRenderer.js`


## How can we verify it:
1. In the DPP repo `fng-cms` change branch to `chore/DPP-1231`
1. On the DPP `package.json` file replace the `@foxdcg/react-select` dependency with this:

```json
"@foxdcg/react-select": "git://github.com/foxbroadcasting/react-select.git#chore/DPP-1231",
```

1. Run `npm i && npm start`
1. Open the DPP on the browser
1. Go to `Types` section
1. Change branch to `dev-chore/DPP-1231`
1. Click on the `+` to open Global Creation flow
1. Select `New Query`
1. Select `Video Queries`
1. Select `None` (it can be any type of Query)
1. Select a panel from the `Assign To Panel` input on the top right corner of the Create a new Video Query screen
1. Search for `FX Presents` and select it
1. Click outside of the Select component and again on the select component
1. Look for `FX Presents` again
1. Notice the selected option is aligned as in the gallery [mocks](https://gallery.io/projects/MCHbtQVoQ2HCZfNvzPdzBSok/files/MCHtA7U1iMGr67g5L1tTjT-XhkieAxJkg6o)
1. Profit 💰 

## Screenshots
![xo9n6ceyxo](https://user-images.githubusercontent.com/1103543/49970669-f0bf5300-fee0-11e8-8292-f02f828c1cc8.gif)



## Related PRs
 


- Type Settings PR: N/A 
- API PR: N/A 

## Checklist
